spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/src/test/java/com/example/TraceSampleApplicationIntegrationTests.java

```diff
@@ -82,17 +95,49 @@
 
   private TraceServiceBlockingStub traceServiceStub;
 
-  @BeforeClass
-  public static void checkToRun() {
-    assumeThat(
-        "Google Cloud Trace integration tests are disabled. "
-            + "Please use '-Dit.trace=true' to enable them. ",
-        System.getProperty("it.trace"),
-        is("true"));
+  private static final String SAMPLE_TOPIC = "traceTopic";
```

Comment:
Since you are creating and deleting the topic for this test now (thank you!!!), could you also make it a unique topic for every run, similar to this: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/5b47cf055ac2d22ba5fb13f81a691b09ca76b70d/spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/pubsub/it/PubSubTemplateIntegrationTests.java#L221

Reply:
Ohhhhhhh. Now I understand why we use UUID thing for this. Thanks!!! Will do this.